portfolio/src/main/java/com/google/sps/servlets/QuoteGenerator.java

```diff
@@ -9,10 +12,38 @@
 /** Handles requests sent to the /quote URL endpoint */
 @WebServlet("/quote")
 public class QuoteGenerator extends HttpServlet {
+  /**
+   * Build the quotes using the author name and quote text.
+   *
+   * @param authorName The author of the quote's name
+   * @param quoteText  The quote text
+   * @return A map with the author's name and the text
+   */
+  private HashMap<String, String> buildQuote(final String authorName, final String quoteText) {
+    final HashMap<String, String> quoteInfo = new HashMap<>();
+    quoteInfo.put("authorName", authorName);
+    quoteInfo.put("quoteText", quoteText);
+    return quoteInfo;
+  }
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>\"Life changes fast. Life changes in the instant.\" - Joan Didion</h1>");
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response)
+      throws IOException {
+    // Add all to the quotes ArrayList
+    final ArrayList<HashMap<String, String>> quotes = new ArrayList<>();
+    quotes.add(buildQuote("Toni Morrision", "Correct what you can; learn from what you can\'t."));
```

Comment:
You are creating a hashmap for each (author name,quote) pair. I don't think you need a hashMap here. You could just have an ArrayList<pair<String,String>> or create new java class called AuthorInfo and have a ArrayList<AuthorInfo>. If in future you need mutiple quotes, you could change the class to have List<String> for quotes. I think this is cleaner than using a hashmap here.

class AuthorInfo {
  String authorName,
  String quote
}

and then

Reply:
Added `Quote` class to simplify code